src/index.html

```diff
@@ -34,10 +33,19 @@
     <script src="//cdnjs.cloudflare.com/ajax/libs/webcomponentsjs/2.0.2/webcomponents-bundle.js"></script>
   
     <style>
-      * {
+      html {
         margin: 0;
         padding: 0;
-        background-color: #020202;
+        background: #fcfff4;
+        /* Old browsers */
```

Comment:
I would remove all of this since all the syntax this project "cares" about is being handled with [PostCSS + Browserslist](https://github.com/ProjectEvergreen/create-evergreen-app/wiki/Build-Pipeline#browserslist).

Reply:
Should we also update README.md screenshot?

Reply:
Good call.  I'll get that setup.